ecommerce/migrations/0020_populate_order_ref_number.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.28 on 2022-08-10 14:45
+
+from django.conf import settings
+from django.db import migrations
+from django.db.models import F, Value
+from django.db.models.functions import Concat
+
+from ecommerce.constants import REFERENCE_NUMBER_PREFIX
+
+
+def populate_order_ref_number(apps, schema_editor):
+    """Populate historical orders' reference numbers"""
+    Order = apps.get_model("ecommerce", "Order")
+
+    Order.objects.filter(reference_number__isnull=True).update(
+        reference_number=Concat(
+            Value(REFERENCE_NUMBER_PREFIX),
+            Value(settings.CYBERSOURCE_REFERENCE_PREFIX),
+            Value("-"),
+            F("id")
```

Comment:
It didn't complain about the missing comma. I had to add it on MITxOnline to pass the python-tests. Looks like there isn't 'black --check' setup on MicroMaster repo